caffe2/serialize/versions.h

```diff
@@ -78,7 +78,7 @@ static_assert(kProducedBytecodeVersion >= kProducedFileFormatVersion,
 // we should support this model_version. For example, we provide a wrapper to
 // handle an updated operator.
 constexpr uint64_t kMinSupportedBytecodeVersion = 0x3L;
-constexpr uint64_t kMaxSupportedBytecodeVersion = 0x5L;
+constexpr uint64_t kMaxSupportedBytecodeVersion = 0x6L;
```

Comment:
we should then probably update also the docs above with each version (currently this only covers v4).